mysql-test/suite/tianmu/r/cast_json_func.result

```diff
@@ -0,0 +1,75 @@
+#
+# Test of function
+# Author: shiz
```

Comment:
rm the pernal msg, including other test files.

Reply:
ok